src/screens/LoginPage/LoginPage.tsx

```diff
@@ -41,6 +41,19 @@ import styles from './LoginPage.module.css';
 import EmailOutlinedIcon from '@mui/icons-material/EmailOutlined';
 import useLocalStorage from 'utils/useLocalstorage';
 
+const socialMediaLinks = [
+  { href: 'https://www.facebook.com/palisadoesproject', logo: FacebookLogo },
```

Comment:
create a constants.ts file in `src/` and add these values over there, and import it from there.

Reply:
Done👍